packages/grid/_modules_/grid/models/api/index.ts

```diff
@@ -3,7 +3,7 @@ export * from './gridParamsApi';
 export * from './gridCoreApi';
 export * from './gridColumnApi';
 export * from './gridDensityApi';
-export * from './gridEditRowApi';
+export * from './gridEditingApi';
```

Comment:
Could we not export `GridEditingSharedApi` / `GridRowEditingApi` / `GridCellEditingApi`

```suggestion
export { GridEditingApi } from './gridEditingApi';
```

We can still access the private api with the full import path.